src/app/_components/RolesList/RolesList.tsx

```diff
@@ -54,7 +56,7 @@ export const RolesList = ({ hideElement = false }: { hideElement?: boolean }) =>
 
             return rolesAndModifiers;
         },
-        [searchValue, typeFilterValue, teamFilterValue]
+        [roles, modifiers, searchValue, typeFilterValue, teamFilterValue]
```

Comment:
Zaczynam się zastanawiać nad sensownością użycia `useMemo` tutaj, ale raczej nie szkodzi. To bardziej komentarz do mnie na przyszłość.